clients/unshaded/src/main/java/tachyon/client/block/TachyonBlockStore.java

```diff
@@ -72,6 +73,8 @@ public BlockInfo getInfo(long blockId) throws IOException {
     BlockMasterClient masterClient = mContext.acquireMasterClient();
     try {
       return masterClient.getBlockInfo(blockId);
+    } catch (TachyonException e) {
+      throw new IOException(e);
```

Comment:
Should we propagate the ConnectionFailedException here instead of wrapping it in IOException? Right now the javadoc is out of date since it says IOException only happens if the block doesn't exist.